libs/application/templates/estate/src/fields/LookupPerson/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { FC, useEffect } from 'react'
+import { useFormContext } from 'react-hook-form'
+import * as nationalId from 'kennitala'
+import { Box, GridColumn, GridRow } from '@island.is/island-ui/core'
+import { FieldBaseProps } from '@island.is/application/types'
+import { InputController } from '@island.is/shared/form-fields'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../lib/messages'
+import { useLazyQuery } from '@apollo/client'
+import { IdentityInput, Query } from '@island.is/api/schema'
+import { IDENTITY_QUERY } from '../../graphql'
+
+export const LookupPerson: FC<FieldBaseProps> = ({ field, error }) => {
+  const { formatMessage } = useLocale()
+  const { id } = field
+  const { setValue, watch, clearErrors } = useFormContext()
+
+  const personNationalId: string = watch(`${id}.nationalId`)
+  const personName: string = watch(`${id}.name`)
+  const e = error as Record<string, string> | undefined
```

Comment:
This is redeclared for the purposes of shortening other LOC like
```typescript
queryError ? e?.name : e?.nationalId ? e?.nationalId : undefined
```
for example? Or does this serve another purpose too?

Just curious.